api/src/Entity/Ndr/AssetProperty.php

```diff
@@ -393,4 +394,16 @@ public function isEqual(AssetInterface $asset)
             $asset->getAddress2() === $this->getAddress2() &&
             $asset->getPostcode() === $this->getPostcode();
     }
+
+    /**
+     * @return float|null
+     */
+    public function getValueTotal()
```

Comment:
Can we not use the PHP `public function getValueTotal(): float | null` here rather than the comment?